source/crud/crud.rst

```diff
@@ -255,7 +255,14 @@ Read
      * @see https://docs.mongodb.com/manual/reference/command/aggregate/
      */
     maxTimeMS: Optional<Int64>;
-    
+
+    /**
+     * The maximum amount of time for the server to wait on new documents to satisfy a tailable cursor
```

Comment:
I think this can borrow some language from the Change Stream spec and state:

```
* @note this option is an alias for `maxTimeMS`, used on `getMore` commands
* @note this option is not set on the `aggregate` command
```

Reply:
👍